cli/Cargo.toml

```diff
@@ -43,8 +43,8 @@ winres = "0.1.11"
 
 [dependencies]
 deno_core = { version = "0.93.0", path = "../core" }
-deno_doc = "0.8.0"
-deno_lint = "0.10.0"
+deno_doc = { git = "https://github.com/denoland/deno_doc" }
+deno_lint = { git = "https://github.com/denoland/deno_lint" }
```

Comment:
This is obviously temporary. We'll need to do a release of these. I'm just opening this as a draft PR then we can convert it to a real PR once we do a release (whenever that happens).

FYI @bartlomieju